src/commands/trace/README.md

```diff
@@ -16,14 +16,18 @@ datadog-ci trace --name "Say Hello" -- echo "Hello World"
 
 - The positional arguments are the command which will be launched and traced.
 - `--name` (default: same as <command>) is a human-friendly name for the reported span.
+- `--tags` is an array of key value pairs of the shape `key:value`. This will be the tags added to the custom span.
+    The resulting dictionary will be merged with whatever is in the `DD_TAGS` environment variable. If a `key` appears both in `--tags` and `DD_TAGS`, whatever value is in `DD_TAGS` will take precedence.
+- `--measures` is an array of key value pairs of the shape `key:value`. This will be the measures added to the custom span.
+    The `value` must be a number.
 - `--no-fail` (default: `false`) will prevent the trace command from failing even when not run in a supported CI Provider. In this case, the command will be launched and nothing will be reported to Datadog.
+- `--dry-run` (default: `false`) it will run the command without sending the custom span. All other checks are performed.
 
 #### Environment variables
 
 Additionally you might configure the `trace` command with environment variables:
 
 - `DD_API_KEY` (**required**): API key used to authenticate the requests.
-- `DD_ENV`: you may choose the environment you want your test results to appear in.
```

Comment:
did we ever read `DD_ENV` ? Are you removing it because it was a mistake?

Reply:
Yes I checked the code and we never read `DD_ENV` so I just removed it from the readme 😬